### DISCUSSION TITLE
p/follow-app-2/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#320] [General]
# p/follow-app-2/

大家好呀，上篇文章给大家推荐了我近期参与内测的一个软件 Follow，得到了大家热情地互动，今天继续给大家深度的分享一下 Follow 的特色功能，同时由于目前 Follow 处于内测邀请阶段，一码处于严重供不应求的状态，我也会给大家推荐下暂时平替的 RSS 开源软件，同时，大家可以关注我，等 Follow 公测开始，我会第一时间告知大家。\n特色功能 RSS 搜索 作为 RSS 阅读器，提供搜索功能，大大降低了大家搜索感兴趣的源的信息，非常便捷方便。\n搜索：可以根据关键词搜索订阅感兴趣的方向的信息源； RSS：可以直接输入 RSS 订阅地址预览订阅； RSSHub：作者与另外一个开源作品组成生态，只要是网站信息开放的，那么"万物皆可 RSS"； 收件箱：你敢相信，也可以接收邮件 📧 ； RSS3：下一代的 RSS 信息订阅方式； 还有 RSS 阅读器的导入功能，你可以快速切换到 Follow 上面来； 谁在阅读 阅读器可以把相同订阅并阅读过这篇文章的网友罗列在文章左上方。\n同时，点击头像会把订阅者自己公开的订阅罗列出来，通过这个功能我补充了好多订阅源 😄\n订阅热点 AI 每天早晚 8 点 AI 基于订阅的源的阅读量来提取 Summary 列表供订阅者阅读。\n生成摘要 AI 生成摘要对我来说很重要，目前我订阅了上百个博客和科技媒体，摘要可以让我快速阅读文章核心内容，并确定是否我感兴趣的文章并继续阅读。\n双语翻译 AI 应该也是通过 AI 实现的自动翻译，可以对比着阅读，订阅英文专业文章也不吃力。\n激励作者 通过区块链技术，发行 POWER 代币，你可以对你感兴趣的文章的作者进行打赏，用来提升作者的影响力和生产出更多高质量的内容，从而实现了良性循环的激励创作体系。\n视频订阅 没错，你可以把 YouTube 或者哔哩哔哩上你感兴趣的 UP 主都订阅起来，他们发表的最新视频就可以通过 Follow 的自动化推送来及时提醒，直接可以在软件内播放，非常地便捷。\n

https://blog.debuginn.com/p/follow-app-2/

**Comment by pangkuku** [comment]
哥，求个邀请码，541985037@qq.com

**Comment by pangkuku** [comment]
哥，邮件我收到了，我能不能预定下一个码，谢谢

**Comment by pangkuku** [comment]
哥，能不能加你一个联系方式，我的v541985037

**Reply by debuginn** [reply]
加你微信了

**Comment by mouse0232** [comment]
follow 我也知道是好东西，但是没有邀请码玩转不了。博主哥能赏一个最好了。233939513@qq.com

**Comment by anchogin** [comment]
大佬，求个邀请码
